SPECS/jasper/jasper.spec

```
@@ -76,23 +110,42 @@ make test -C builder
 %{_libdir}/libjasper.so
 %{_libdir}/pkgconfig/jasper.pc
 
+%post libs -p /sbin/ldconfig
+%postun libs -p /sbin/ldconfig
+
```

Since this is 2.0, we can actually finally use the scriptlets:

```suggestion
%ldconfig_scriptlets libs

```

---

Done.